eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -73,14 +73,14 @@ jobs:
       - ${{ each artifact in parameters.Artifacts }}:
         - script: |
             python3 --version
-            python3 eng/versioning/set_versions.py --build-type ${{parameters.SDKType}} --build-qualifier dev.$(Build.BuildNumber) --artifact-id ${{artifact.name}} --group-id ${{artifact.groupId}}
-          condition: eq(variables['SetDevVersion'],'true')
+            python3 eng/versioning/set_versions.py --build-type ${{parameters.SDKType}} --update-type all --build-qualifier dev.$(Build.BuildNumber) --artifact-id ${{artifact.name}} --group-id ${{artifact.groupId}}
+          #condition: eq(variables['SetDevVersion'],'true')
```

Comment:
@mitchdenny I'm not sure exactly what you're trying to do but this change isn't correct. The update-type argument is **library**, **external_dependency** or **all** (probably should have been _both_ instead of _all_). set_versions is a script that **only** manipulates library versions which means that the update-type argument was completely unnecessary and was removed from the script. Similarly, the call below to update_versions, was correct with the update_type of library since, the previous set_versions is only ever manipulating library versions.

I'm assuming the commented out conditions were for testing purposes and not part of this PR?